docs/resources/fgs_function.md

```diff
@@ -53,50 +53,138 @@ EOF
+  If omitted, the provider-level region will be used. Changing this creates a new Function resource.
+
+* `name` - (Required, String, ForceNew) Specifies a unique name for the function.
+
+* `app` - (Required, String) Specifies group to which the function belongs.
+
+* `handler` - (Required, String) Specifies entry point of the function.
+
+* `memory_size` - (Required, Int) Specifies memory size(MB) allocated to the function.
+
+* `runtime` - (Required, String) Specifies environment for executing the function.
+
+* `timeout` - (Required, Int) Specifies timeout interval of the function, ranges from 3s to 900s.
+
+* `code_type` - (Required, String) Specifies function code type, which can be inline: inline code, zip: ZIP file,
+	jar: JAR file or java functions, obs: function code stored in an OBS bucket.
+
+* `func_code` - (Optional, String) Specifies function code. When code_type is set to inline, zip, or jar, this parameter is mandatory,
+	and the code can be encoded using Base64 or just with the text code.
+
+* `code_url` - (Optional, String) Specifies the code url. This parameter is mandatory when code_type is set to obs.
+
+* `code_filename` - (Optional, String) Specifies the name of a function file, This field is mandatory only when coe_type is
+	set to jar or zip.
+
+* `depend_list` - (Optional, String) Specifies the dependencies of the function.
+
+* `user_data` - (Optional, String) Specifies the Key/Value information defined for the function.
+
+* `agency` - (Optional, String) Specifies the agency. This parameter is mandatory if the function needs to access other cloud services.
+
+* `app_agency` - (Optional, String) Specifies An execution agency enables you to obtain a token or an AK/SK for accessing other cloud services.
 
-* `name` - (Required, String, ForceNew) A unique name for the function. Changing this creates a new function.
+* `description` - (Optional, String) Specifies the description of the function.
 
-* `app` - (Required, String) Group to which the function belongs. Changing this creates a new function.
+* `initializer_handler` - (Optional, String) Specifies the initializer of the function.
 
-* `code_type` - (Required, String, ForceNew) Function code type, which can be inline: inline code, zip: ZIP file,
-	jar: JAR file or java functions, obs: function code stored in an OBS bucket. Changing this
-	creates a new function.
+* `initializer_timeout` - (Optional, Int) Specifies the maximum duration the function can be initialized. Value range: 1s to 300s.
 
-* `code_url` - (Optional, String, ForceNew) This parameter is mandatory when code_type is set to obs. Changing this
-	creates a new function.
+* `enterprise_project_id` - (Optional, String, ForceNew) Specifies the enterprise project id of the function.
+  Changing this creates a new function.
 
-* `description` - (Optional, String, ForceNew) Description of the function. Changing this creates a new function.
+* `vpc_id`  - (Required, String) Specifies the ID of VPC.
```

Comment:
seems `vpc_id` and `network_id` are optional

Reply:
fixed